lib/actions/deploy.coffee

```diff
@@ -58,6 +63,15 @@ performUpload = (image, token, username, url, size, appName) ->
 
 	uploadToPromise(post, size)
 
+uploadLogs = (logs, token, url, buildId, username, appName) ->
+	request = require('request')
+	url = url || process.env.RESINRC_RESIN_URL
```

Comment:
That's weird, `require('resin-settings-client').get('resinUrl')` is where the `url` value comes from, but I was experiencing weirdness where when setting the `RESINRC_RESIN_URL` env var to `resindev.io` that it wasn't being mirrored in the settings call. I've just re-tested and it is now, so will change.